build.py

```diff
@@ -363,6 +363,43 @@ def run_python_integration_tests(cwd, interpreter):
     )
     step_end()
 
+if build_samples:
```

Comment:
As of https://github.com/microsoft/qsharp/pull/1739 all samples testing now occurs via `cargo test` and auto-generated Rust unit tests, so this section was deleted. I think the automatic merge may have reintroduced it. You can delete it here and confirm the added project is automatically picked up and compiled as part of the Rust tests.